Formula/qpid-proton.rb

```diff
@@ -0,0 +1,47 @@
+class QpidProton < Formula
+  desc "High-performance, lightweight AMQP 1.0 messaging library."
+  homepage "https://qpid.apache.org/proton/"
+  url "https://www-us.apache.org/dist/qpid/proton/0.17.0/qpid-proton-0.17.0.tar.gz"
+  sha256 "6ffd26d3d0e495bfdb5d9fefc5349954e6105ea18cc4bb191161d27742c5a01a"
+  head "git://git.apache.org/qpid-proton.git"
+
+  depends_on "cmake" => :build
+  depends_on "libuv" if build.head?
```

Comment:
Use a `head do` block instead of this style. Plenty of examples around, such as [this one](https://github.com/Homebrew/homebrew-core/blob/09b8e6ce09932a2394e83edee996e0a62fdb179d/Formula/frobtads.rb#L13-L19).